src/cli/load.js

```diff
@@ -105,6 +112,14 @@ const load: Command = async (args, std) => {
         plugin = arg;
         break;
       }
+      case "--organization": {
```

Comment:
Before this commit, the `cli/` package had 100% test coverage. Please
maintain that.